### ISSUE TITLE
Multiremote support

### ISSUE THREAD

**Issue by db798** [webdriverio-community/wdio-wait-for#5]
I've just started looking at converting some protractor tests to webdriver.io and I'm trying to use this package to make the switchover as easy as possible. 

Our tests use multiple browsers `Multiremote` and it looks like this package may not handle that as expected.

In my `wdio.conf` file I have this capabilities configuration

```
capabilities: {
        browser1: {
            capabilities: {
                browserName: 'chrome'
            }
        },
        browser2: {
            capabilities: {
                browserName: 'chrome'
            }
        }
    }
```

In my test, I am navigating to a url with one of the browsers and trying to wait until the title is correct. 

```
await browser1.url('https://webdriver.io/docs/wdio-wait-for/#api');

await browser1.waitUntil(
   titleContains('expected conditions'), {
   timeout: 5000,
   timeoutMsg: 'expected title....'
});
```

The above snippet does not work, the test timeouts. But using the standard waitUntil, the below does work.

```
await browser1.url('https://webdriver.io/docs/wdio-wait-for/#api');

await browser1.waitUntil(
  async function () {
    const title = await browser1.getTitle();

    return title.includes('expected conditions')
  },
  {
    timeout: 5000,
    timeoutMsg: 'expected title....'
  }
);
```

It seems that when `Multiremote` is used the wait is waiting for both browsers.

Are my expectations/assumptions correct here, is this something that could be supported?

**Comment by elaichenkov** [comment]
Hey!
Thanks for the well-described report!
Yeah, you're right this package can't handle Multiremote now, but I can release a new version with the additional parameter to fix it.

For instance:

```javascript
 await browser1.waitUntil(
   //                                    ↓ additional argument (browser instance)
   titleContains('expected conditions', browser1), {
   timeout: 5000,
   timeoutMsg: 'expected title....'
 });
```

What do you think about that?

**Comment by christian-bromann** [comment]
if we use the API as follows:

```js
await browser1.waitUntil(
   titleContains('expected conditions'),
   {
      timeout: 5000,
      timeoutMsg: 'expected title....'
   }
);
```

then I assume `titleContains` will be called with the scope of `browser1`. So if we just use `this` it should work out of the box, no?

**Comment by elaichenkov** [comment]
Hi @christian-bromann 
Yep, make sense. I'll refactor it. Thank you!

**Comment by elaichenkov** [comment]
Hey @db798 
I've just published the `2.1.0` version. Can you please verify it?
